src/billing/api/index.ts

```diff
@@ -32,8 +32,8 @@ const makeResponse = (status, data, ...args) => {
 export const getAccount = (): ReturnType<typeof apiGetAccount> => {
   const account: Account = {
     id: 'account_1',
-    marketplace: 'AWS',
-    type: 'free',
+    marketplace: 'gcp',
```

Comment:
Btw valid marketplaces are `aws`, `gcm`, and `azure`.

Reply:
👍🏽 I'm gonna add a type to the yml to track those changes